dev-lang/php/php-7.4.30.recipe

```diff
@@ -50,44 +50,41 @@ REQUIRES="
 	lib:libicuuc$secondaryArchSuffix
 	lib:libjpeg$secondaryArchSuffix
 	lib:libnghttp2$secondaryArchSuffix
-	lib:libonig$secondaryArchSuffix
 	lib:libpcre$secondaryArchSuffix
 	lib:libpng16$secondaryArchSuffix
 	lib:libsqlite3$secondaryArchSuffix
 	lib:libssl$secondaryArchSuffix
-	lib:libwebp$secondaryArchSuffix
 	lib:libxml2$secondaryArchSuffix
 	lib:libz$secondaryArchSuffix
-	lib:libzip$secondaryArchSuffix
+	lib:libwebp$secondaryArchSuffix
+	lib:libonig$secondaryArchSuffix
```

Comment:
Why were these moved? They need to be alphabetized.

Reply:
Patches were regenerated after a successful build.

Reply:
Fixed